src/lib.rs

```diff
@@ -554,6 +563,21 @@ impl Contract {
             .set(json!({ get_devhub_community_account(&handle): data }));
     }
 
+    pub fn create_discussion(&mut self, handle: CommunityHandle, data: Value) {
+        require!(env::prepaid_gas() >= CREATE_DISCUSSION_GAS, "Require at least 60 Tgas");
+        // Post the discussion on the user social account
+        let promise = social_db_contract()
+            .with_unused_gas_weight(1)
+            .set(json!({ format!("{}", env::predecessor_account_id()): data }));
+
+        // Post to discussions account
+        promise.then(
+            social_db_contract()
+                .with_unused_gas_weight(1)
+                .set(json!({ get_devhub_discussions_account(&handle): data })),
+        );
```

Comment:
We don't want to post a duplicate post. We need to reshare post using NEAR Social native reshare functionality, see the reshared posts here: https://near.social/mob.near/widget/ProfilePage?accountId=zavodil.near

<img width="411" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-contract/assets/304265/7b9876b4-5aba-452f-8478-975094c2b918">